EdFi.Ods.Utilities.Migration.Tests/appsettings.pgsql.json

```diff
@@ -1,12 +1,17 @@
 {
   "ConnectionStrings": {
-    "EdFi_Ods_Migration_Test": "host=localhost;port=5432;username=postgres;database=EdFi_Ods_Migration_Test;pooling=true;Minimum Pool Size=10;Maximum Pool Size=50;client encoding=utf-8;command timeout=300",
+    "EdFi_Ods_Migration_Test": "host=localhost;port=5432;username=postgres;database=EdFi_Ods_Migration_Test;pooling=true;Minimum Pool Size=10;Maximum Pool Size=50;client encoding=utf-8;command timeout=3000",
```

Comment:
what is causing this change to 50 minutes?

Reply:
Postgres Integration Test will require long time for Glendale Tests local testing otherwise getting time out error ,So  I increased it here .

Reply:
I agree with you . we add note in the documentation for this to modify the connection strings command timeout to 3000